src/NodaTime/project.json

```diff
@@ -13,6 +13,12 @@
     "requireLicenseAcceptance": false,
     "repository": {
       "url": "https://github.com/nodatime/nodatime.git"
+    },
+    "files": {
+      "mappings": {
+        "build/": "../../packaging/NodaTime.targets",
+        "lib/native/": [ "../../packaging/_._" ]
```

Comment:
Yes, we need this, because the `_._` creates an empty directory (`lib/native`), which is found/used by NuGet while installing the package. Otherwise, the package cannot be installed to a C++/CLI project. The TFM for C++ and C++/CLI is both `native`.